src/common/engine.py

```diff
@@ -325,6 +328,20 @@ def _recover_previous_version(
                     shutil.copy(src_file, tournament.file)
                     logger.debug('%s > %s', str(src_file), str(tournament.file))
                     tournaments_number += 1
+        logger.info('Recovering misc files...')
+        files_to_recover: list[Path] = [
+            TMP_DIR
+            / f'{FideDatabase.static_id()}].{SharlyChessConfig.federation_database_ext}',
+        ] + list(
+            itertools.chain.from_iterable(plugin_manager.hook.get_files_to_recover())
+        )
```

Comment:
```suggestion
        files_to_recover: list[Path] = [
            database.file for file in
            LocalSourceDatabaseManager.objects()
        ]   
```
There already is a way to list source databases, no need to use another hook